data/remnant missions.txt

```diff
@@ -495,3 +560,772 @@ mission "Remnant: Technology Available"
+	shipyard "Remnant"
+		"Albatross"
+		"Pelican"
+
+
+
+mission "Remnant: Salvage 4"
+	source "Viminal"
+	name "Track down the <npc>"
+	description "Track down and disable a Korath ship that escaped the raid on Viminal."
+	to offer
+		has "Remnant: Salvage 3: done"
+	on offer
+		conversation
+			`Walking into the cafeteria you immediately sense the buoyant mood. According to a report on the video screen very few Remnant ships were actually lost. Furthermore, of the ships that were lost, the escape pods worked perfectly, ensuring none of the crew were killed. Groups of crew are gathered around tables intermingled with the ground teams that keep their ships operational, and based on the snippets you can catch, they seem to be trading stories of crack shots and clever improvisations.`
+			`	Tali sits over at a corner booth with the prefect who met you at the dock and a couple other mechanics, and she waves you over. "Captain <last>! So, how do you like fighting with our fleet?" asks one of the mechanics.`
+			choice
+				`	"It seems odd to have a planetary defense fleet plundering."`
+				`	"It was a tough battle."`
+					goto next
+	
+			`	"I suppose it would," chants one of them. "We need access to their tech, though. Otherwise we'll fall behind." Judging by the finality of his statement, this is held to be common knowledge among the Remnant.`
+				goto next
+	
+			label next
+			`	While you're conversing with the group, the prefect suddenly glances down at his comlink, reads a message, then looks up at you. "It appears that a Korath ship survived the raid and has destroyed a few ships without taking cargo. Could you deal with it?`
+			choice
+				`	"Sure."`
+				`	"Sorry, I don't have time to help you out."`
+					decline
+
+			`	"Thank you," he signs. "It is acting differently, jumping around destroying ships instead of looting and running. If it simply looted and left we would be pleased, but now it is learning about us instead, so it has reached the end of its usefulness. If you could disable it so that our technicians can get onboard, it would be much appreciated." He pauses, then continues, "I doubt we'll be able to salvage the ship itself - they pack those things with soldiers - but we should be able to strip everything else of use off it."`
+				accept
+	npc disable
+		government Korath
+		personality heroic vindictive target uninterested waiting
+		system
+			distance 1 2
+		fleet
+			names "korath"
+			cargo 3
+			variant
+				"Korath Raider (Hyperdrive)"
+		dialog "You have disabled the Korath ship that was left over from the raid on <planet>. You can now return there to collect your payment."
+	on complete
+		payment 500000
+		dialog `You are met by the prefect who thanks you for disabling it and hands you a credit chip for <payment>. "I think Tali might have some more work for you in the spaceport, but in the meantime, don't forget to drop off any salvaged outfits in the outfitters so the researchers can look at them."`
```

Comment:
For the sake of grammatical accuracy we should mention early on who is speaking this line. It's fairly obvious that it's the prefect, but only in hindsight.

Reply:
Should I specify "the military prefect?" or something to that effect?